database/insertData/03_template_E_drugRegistration.js

```diff
@@ -556,11 +566,11 @@ exports.queries = [
               # Apply General
               { permissionNameId: 10100 }
               # Review Drug Registration Stage 1
-              { permissionNameId: 10300, restrictions: {canSelfAssign: true}, level: 1, stageNumber: 1 }
+              { permissionNameId: 10300, restrictions: {canSelfAssign: true}, levelNumber: 1, stageNumber: 1 }
               # Assign Drug Registration Stage 2
-              { permissionNameId: 10200, level: 1, stageNumber: 2 }
+              { permissionNameId: 10200, levelNumber: 1, stageNumber: 2 }
               # Review Drug Registration Stage 2 
-              { permissionNameId: 10400, level: 1, stageNumber: 2 }
+              { permissionNameId: 10400, levelNumber: 1, stageNumber: 2 }
```

Comment:
Maybe should remove this while we don't have a 2nd level review for `stageNumber`  2 yet - I'll add this to be done as part of changes required to finish up the Drug Registration template.

Reply:
Okay, have commented out this line for now: e17d0baf